styles/mixins/bg-colors.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = function (config) {
+  return function (mixin) {
```

Comment:
I like the idea, but I think we can be a little bit more explicit.  In the color object, lets add `background-X` and then have `.bg-X` auto add that background.  If we want to share colors just use something like:

```color:
  background-body: '{{color.someOtherVar}}'
```